phalcon/config.zep

```diff
@@ -50,6 +50,10 @@ use Phalcon\Config\Exception;
 class Config implements \ArrayAccess, \Countable
 {
 
+	protected static _pathDelimiter;
```

Comment:
Please don't introduce new static properties, you can use object instance to get properties